### ISSUE TITLE
ShowRecuitInfoInsideZoneTooltip broken in 1.11

### ISSUE THREAD

**Issue by paulgrove** [Mad-Mods-Phoenix-Point/AssortedAdjustments#6]
AssortedAdjustments: 1.10.1.4
Polaris 1.11 EGS
Modnix 2.5.7

Tooltip is not replaced:
![image](https://user-images.githubusercontent.com/184906/119736058-181df680-be75-11eb-834d-ff1cedf8af21.png)

```
  "EnableUIEnhancements": true,
  "ShowDetailedResearchAndProduction": true,
  "ShowRecruitInfoInsideZoneTooltip": true,
  "ShowExtendedBaseInfo": true,
  "ShowExtendedHavenInfo": true,
  "PersistentClassFilter": true,
  "PersistentClassFilterInitDisabled": false,
  "HideSiteAddons": false,
  "CompressPopulationBar": true,
  "HidePopulationBar": false,
  "BetterObjectives": true,
```

**Comment by paulgrove** [comment]
I miss understood this feature - I can see the Traits in the Geoscope hover

**Comment by paulgrove** [comment]
Ok I'm tired, I guess I was right first time - this is broken as above with `"ShowRecruitInfoInsideZoneTooltip": true,`

**Comment by mad2342** [comment]
Fixed with 1.11.0.0
Closing this issue, feel free to open another if problems persist.
